packages/integration-tests/utils/fixtures.ts

```diff
@@ -0,0 +1,44 @@
+import { test as base } from '@playwright/test';
+import fs from 'fs';
+import path from 'path';
+
+import { generatePage } from './generatePage';
+
+const getAsset = function(assetDir: string, asset: string): string {
+  if (fs.existsSync(`${assetDir}/${asset}`)) {
+    return `${assetDir}/${asset}`;
+  } else {
+    return `${path.dirname(assetDir)}/${asset}`;
+  }
+};
```

Comment:
```suggestion
const getAsset = (assetDir: string, asset: string): string => {
  const path = `${assetDir}/${asset}`;
  if (fs.existsSync(path)) {
    return path;
  }
  
  return `${path.dirname(assetDir)}/${asset}`;
};
```

Would also be open to making `existsSync` async